site/content/en/docs/talks_and_presentations/_index.md

```diff
@@ -9,6 +9,17 @@ description: >
   Talks and presentations showcasing Kueue
 ---
 
+# 2025
+
```

Comment:
Do you have youtube links?

Reply:
KubeCon hasn't published them yet.... we can either wait to merge this until they do or we can update it later once KubeCon publishes the videos.

Reply:
I'm ok either way. Actually, it could be helpful to get both the youtube and sched links, since sched has more metadata like the abstract.

Reply:
+1 on both Youtube and sched

Reply:
I changed to use the new format from #4941  -- I think it is another week until the videos from London will be published.